recipe/meta.yaml

```
@@ -80,6 +72,7 @@ outputs:
         - xarray
         - scipy
         - pooch
+        - matplotlib >2.2
```

```suggestion
        - matplotlib-base >2.2
```
A lightweight part of the `matplotlib` package (without `pyqt`)